docs/operator/k8s_guide.md

```diff
@@ -0,0 +1,725 @@
+- The `RUCIO_SSL_PROTOCOL` variable must be explicitly set.
+
+Please notice that in order to have LBs produced, ***the Helm chart must be applied***. 
+This will come with several errors related to certificates, that will be fixed in the next steps.
+### Setup the LB
+The minimal configuration for LBaaS is the following:
+```yaml
+    service:
+      type: LoadBalancer
+      port: 443
+      targetPort: 443
+      protocol: TCP
+      name: https
+```
+Once the Helm chart is applied, it will trigger the `openstack-cloud-controller-manager` pod, that will automatically instantiate the requested LBs. 
+
+Please notice that in order to have LBs, one must request a quota change to the openstack project:
+![[/img/get-lbs.png]]
+
+To check the status of the LBs, the CLI command can be used: 
+```sh
+openstack loadbalancer list
+```
+
+Then, a few attributes must be changed in order to obtain the necessary DNS:
+```sh
+openstack loadbalancer set --tag landb-set="RUCIO-IT" <lb-ID>
+openstack loadbalancer set --tag landb-alias="compass-rucio.cern.ch" <lb-ID>
+openstack loadbalancer set --description="compass-rucio.cern.ch" <lb-ID>
+openstack loadbalancer set --tag main-user="rucio-it-admins" <lb-ID>
+```
+
+Now, the LB must be added to the LanDB set from the dashboard, under IP Addresses.
+To do that, first retrieve the `vip_address` of the LB:
+```sh
+ openstack loadbalancer show <lb-ID> | grep vip_address
+```
+
+Then, retrieve the pointer to the LB:
+
+```sh
+host <vip_address>
+222.xx.xxx.xxx.in-addr.arpa domain name pointer lbaas-xxx4a6c2-xxxx-xxxx-xxxx-59489ffd7xxx.cern.ch.
+```
+
+Finally, add `lbaas-...cern.ch` as an IP address in the LanDB set dashboard. 
+
+### Produce the certificates related to the landb-alias
+Read the [help page](https://ca.cern.ch/ca/Help/?kbid=021002) about Grid Host certs
+
+Then, request a cert using [dedicated page](https://ca.cern.ch/ca/host/HostSelection.aspx?template=EE2Host&instructions=auto). 
+In the current example, a cert for `compass-rucio.cern.ch` (corresponding to the LB `landdb-alias`) was requested:
+
+> [!tip] Certs pro move
+> Create a [grid Host certificate](https://ca.cern.ch/ca/host/Request.aspx?template=ee2host) linked to both servers and auth domains. Use the SAN box add as many domains as you want. Then you'll have one cert to rule them all:
+ 
+![[/img/grid-host-certs.png]]
+
+#### Create the `host`, `key`, `ca` and `GridCA` files
+
+As recommended in the [documentation](https://ca.cern.ch/ca/Help/?kbid=024100), execute:
+```sh
+openssl pkcs12 -in compass-rucio.p12 -clcerts -nokeys -out hostcert.pem
+
+openssl pkcs12 -in compass-rucio.p12 -nocerts -nodes -out hostkey.pem
+
+chmod 400 hostkey.pem 
+chmod 644 hostcert.pem 
+```
+These files have to be stored in the same path as described by the `RAW_SECRETS_SERVERS` variable used in the secrets creation.
+##### Important information about `ca.pem` and `GridCA.pem`
+An additional file, `ca.pem` (i.e. `CERN-bundle.pem`) must be created. 
```

Comment:
What happens if they aren't created? What do they do? (another question for a non-cern user trying to find their equivalent service)

Reply:
Explained the purpose of the CA files. 
If needed, I can also explain what happens if they are not created; I didn't do it now because think it's reasonably clear given the added text.